code/modules/surgery/organs/internal/heart/heart_ethereal.dm

```diff
@@ -1,6 +1,7 @@
 /obj/item/organ/internal/heart/ethereal
 	name = "crystal core"
-	icon_state = "ethereal_heart" //Welp. At least it's more unique in functionaliy.
+	icon_state = "ethereal_heart-on" //Welp. At least it's more unique in functionaliy.
```

Comment:
should we remove this comment i have no idea what it refers to. unless you think it's sovl